src/components/Form.js

```diff
@@ -0,0 +1,38 @@
+import React from "react";
+
+
+function Form({setStatus, setInputText, todos, setTodos, inputText}) {
+    
+    const inputTextHandler = (e) => {
+        setInputText(e.target.value);
+     };
+
+     const submitTodoHandler = (e) => {
+         e.preventDefault();
+         setTodos([
```

Comment:
I think you should also check when `inputText` has a value to avoid adding an empty todo.

```js
if (inputText) {
  setTodos()
}
```